frontend/src/App.tsx

```diff
@@ -1,12 +1,9 @@
 import "./App.css";
 import { FormEvent, useCallback, useEffect, useState } from "react";
```

Comment:
keep relative imports at the bottom and imports from ract other npm libraries at the top

Reply:
Just to expand on this, a practice we try to follow is to keep imports from npm modules separated from those which are from our own code.

In this case, we would like to keep the line `import { FormEvent, useCallback, useEffect, useState } from "react";` at the top, followed by a blank line. Then we would add all imports from our own code, like `import "./App.css";`.

Doing this makes it easy to understand which libraries a specific file/component depends on. Check this for all the files.